gravatar/src/main/java/com/gravatar/restapi/apis/AvatarsApi.kt

```diff
@@ -28,9 +28,7 @@ internal class AvatarsApi(
 ) : ApiClient(basePath, client) {
     internal companion object {
         @JvmStatic
-        val defaultBasePath: String by lazy {
-            System.getProperties().getProperty(ApiClient.BASE_URL_KEY, "https://api.gravatar.com/v3")
```

Comment:
As mentioned in the git commit message. This was breaking the Compose preview feature and because we are not using it I simply removed the `System.getProperties` from the `.mustache`.